src/pages/profiles/[userId].tsx

```diff
@@ -204,6 +205,8 @@ function Picture({ userId, profile, updateProfile, SaveButton }: {
   SaveButton: React.ComponentType,
 }) {
   const [me] = useUserContext();
+  const [isModalOpen, setModalOpen] = useState(false);
+  const [imageUrl, setImageUrl] = useState(profile.照片链接); 
```

Comment:
Don't use this extra variable. Read image url directly from the profile parameter.